paddle/contrib/inference/paddle_inference_api.h

```diff
@@ -12,49 +12,58 @@
    See the License for the specific language governing permissions and
    limitations under the License. */
 
+/*
+ * This file contains the definition of a simple Inference API for Paddle.
+ *
+ * ATTENTION: It requires some C++ features, for lower version C++ or C, we
+ * might release another API.
+ */
+
 #pragma once
 
+#include <memory>
 #include <string>
 #include <vector>
 
-namespace paddle {
+struct PaddleTensor {
```

Comment:
why no namespace?

Reply:
the namespace will make the symbol name more complex if user use `dlsym` to load the APIs from a dynamic library.

Just add `paddle` prefix to each API, and the most important method `CreatePaddlePredictor` can load easily by `dlsym`.